cypress/e2e/navigation/to-transactions.cy.js

```diff
@@ -1,4 +1,4 @@
-suite(["@CI"],'transaction spotlight', () => {
```

Comment:
I notice that this page sometimes fails on the graphql end. Sometimes it doesn't. I think we should disable this test for now and bank on the new indexer-api being performant.

Reply:
https://github.com/Granola-Team/mina-block-explorer/issues/368